docs/source/intervention_models/lipid_based_nutrient_supplements/index.rst

```diff
@@ -0,0 +1,267 @@
+more than 280 kcal/day (> 90 g dose).LNS are nutrient dense, require no cooking before use, and can be stored for months even in warm conditions. 
+[DAS_Cochrane_Review_2019]_
+
+:underline:`Alternative recipes and formulations, other than LNS`
+
+Alternative recipes and formulations, other than LNS, are currently
+being explored using cereals mixed with other ingredients, including
+whey, soy protein isolate, dried skimmed milk, and sesame,
+cashew and chickpea paste, among others. These are
+fortified with vitamins and minerals and are commonly called fortified
+blended foods (FBF). An example of a commonly used FBF
+is corn soy blend plus (CSB ++), which is a cooked blend of milled,
+heat-treated corn and soybeans that is fortified with a vitamin and
+mineral premix. Multiple micronutrient powders (MNP) are also an
+alternative way of providing micronutrients. These are single-dose
+packets of vitamins and minerals in powder form that can be sprinkled
+onto any ready to eat semi-solid food consumed at home,
+school or any other point of use. [DAS_Cochrane_Review_2019]_
+
+.. todo::
+
+   Add a general narrative overview of the intervention, including what it is, what outcomes it affects, if/how/when/where it has been used, etc.
+
+.. todo::
+
+  Fill out the following table with a list of known outcomes affected by the intervention, regardless of if they will be included in the simulation model or not, as it is important to recognize potential unmodeled effects of the intervention and note them as limitations as applicable.
+
+  The table below provides example entries for large scale food fortification with iron.
+
+.. list-table:: Affected Outcomes
+  :widths: 15 15 15 15
+  :header-rows: 1
+
+  * - Outcome
+    - Effect
+    - Modeled?
+    - Note (ex: is this relationship direct or mediated?)
+  * - Hemoglobin concentration
+    - Increases population mean
+    - Yes
+    - 
+  * - Malaria
+    - Increases incidence rate
+    - No
+    - 
+
+Baseline Coverage Data
+++++++++++++++++++++++++
+
+.. todo::
+
+  Document known baseline coverage data, using the table below if appropriate
+
+.. list-table:: Baseline coverage data
+  :widths: 15 15 15 15 15
+  :header-rows: 1
+
+  * - Location
+    - Subpopulation
+    - Coverage parameter
+    - Value
+    - Note
+  * - 
+    - 
+    - 
+    - 
+    - 
+
+Vivarium Modeling Strategy
+--------------------------
+
+.. todo::
+
+  Add an overview of the Vivarium modeling section.
+
+.. todo::
+
+  Fill out the following table with all of the affected measures that have vivarium modeling strategies documented
+
+.. list-table:: Modeled Outcomes
+  :widths: 15 15 15 15 15 15 15
+  :header-rows: 1
+
+  * - Outcome
+    - Outcome type
+    - Outcome ID
+    - Affected measure
+    - Effect size measure
+    - Effect size
+    - Note
+  * - Lung cancer
```

Comment:
This is a surprise to me, so I'm checking if you intend to include it

Reply:
ah, these are the examples from the template...I've left them in there and will revise as I build out the document.